deploy/helm/flightctl/templates/alertmanager/flightctl-alertmanager-proxy-deployment.yaml

```diff
@@ -91,6 +91,14 @@ spec:
               name: flightctl-alertmanager-proxy-openshift-token
               readOnly: true
             {{- end}}
+            - mountPath: /root/.flightctl/certs/server.crt
+              name: flightctl-api-server-tls
+              subPath: tls.crt
+              readOnly: true
+            - mountPath: /root/.flightctl/certs/server.key
+              name: flightctl-api-server-tls
+              subPath: tls.key
+              readOnly: true
```

Comment:
Also here the alertmanager-proxy would ideally have its own server cert, but let's fix that later.